commonknowledge/mapped
# fix: makes some copy more consistent and fixes some hot reload issues

## Description
- Makes Reports vs Maps consistent (uses Maps)
- Makes Data / Data Sources / Member list / Membership lists consistent (uses Membership lists)
- Adds copy to explain Custom Data Sources
- Improves UX of "Enrichment" section of Data Source page
- Also fixes hot reloading issues caused by component files exporting more than just the component – this breaks hot reload

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-316/improve-consistency-in-ui-labels-names

## How Can It Be Tested?
Check out the branch and verify that the labels are consistent across the Map pages and the Data Sources pages.
